README.md

```diff
@@ -0,0 +1,74 @@
+![image](https://img.shields.io/pypi/dm/discord-py-slash-command.svg)
+![image](https://img.shields.io/pypi/pyversions/discord-py-interactions.svg)
+![image](https://img.shields.io/pypi/v/discord-py-interactions.svg)
+![image](https://readthedocs.org/projects/interactionspy/badge/?version=latest)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+![image](https://discord.com/api/guilds/789032594456576001/embed.png)
+
+------------------------------------------------------------------------
+
+# Interactions.py - A Feature-rich Discord Bot Framework for Python
+
+A highly extensible, easy to use, and feature complete bot framework for Discord, `Interactions.py` is the culmination of years of experience in the Discord API and bot development. This library has been built from the ground up with community feedback and suggestions in mind, providing a modern and intuitive API for easy interaction with the Discord API in Python.
+
+## Key Features
+Interactions.py offers a range of features for building Discord bots with Python, including:
```

Comment:
```suggestion
interactions.py offers a wide range of features for building Discord bots and web applications alike:
```

Reply:
This change is for SEO. The word python needs to stay. Happy to expand the usecase, but that word needs to stay

Reply:
Readability change

`interactions.py offers a wide range of features for building Python-powered Discord bots and web applications alike:`